code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -151,7 +151,7 @@
 	if(!.)
 		return
 	var/mob/living/carbon/human/H = user
-	if(!istype(H) || !H.dna || !H.dna.species || !H.dna.species.can_wag_tail(H))
+	if(!istype(H) || !H.dna || !H.dna.species || !H.dna.species.can_wag_tail(H)) //Monkestation edit
```

Comment:
What is the difference here that required the edit comment?

Reply:
There is none, change was made elsewhere, deleting